### PR TITLE
feat(misc.): add canonical WORKFLOW_RELEASE version

### PR DESCRIPTION
**Pull request by vdice**
So that update for new release need only happen in one place.

TODO:
- [x] update `component_release.groovy` to be added in https://github.com/deis/jenkins-jobs/pull/129

Ref https://github.com/deis/workflow/pull/363

### CODE REVIEW

File: jobs/workflow_test_release.groovy [hunk 1]
```diff
@@ -40,17 +50,17 @@ job(name) {
...
+    stringParam('RELEASE', defaults.workflow.release, "Release string for resolving workflow-[release](-e2e) charts")
+    stringParam('HELM_REMOTE_REPO', defaults.helm["remoteRepo"], "The remote repo to use for fetching charts.")
     stringParam('E2E_RUNNER_IMAGE', 'quay.io/deisci/e2e-runner:canary', "The e2e-runner image")
     stringParam('E2E_DIR', '/home/jenkins/workspace/$JOB_NAME/$BUILD_NUMBER', "Directory for storing workspace files")
     stringParam('E2E_DIR_LOGS', '${E2E_DIR}/logs', "Directory for storing logs. This directory is mounted into the e2e-runner container")
   }
 
   triggers {
-    cron('@daily')
+    githubPush()
```
Comment (mboersma) [inline jobs/workflow_test_release.groovy:63]: An unrelated change here. No need to split the commit, but could you explain this briefly @vdice? I assume daily was the wrong trigger.

Comment (vdice) [inline jobs/workflow_test_release.groovy:63]: Thanks for keeping me honest, @mboersma !  Indeed, strictly speaking, this should ideally have been a separate change.  However, to explain my reasoning:  

I wanted the release test job to trigger off of push events to release branches, rather than arbitrarily running them daily.  The real value of this job is during release time and I don't think there is value in running them arbitrarily during inter-release iterations -- though definitely receptive to feedback on this assumption.

Comment (mboersma) [inline jobs/workflow_test_release.groovy:63]: > The real value of this job is during release time and I don't think there is value in running them arbitrarily

I agree with that. Thanks for the explanation.

**Comment by arschles** [comment]
LGTM, pending the unchecked TODO in the PR description
